sawaliram_auth/templates/sawaliram_auth/base.html

```diff
@@ -16,11 +16,12 @@
     </head>
     <body>
         <div class="container auth-container">
+            {% include 'snippets/show_messages.html' %}
```

Comment:
Messages are not shown really in auth views. I think we can stick to the form error messages with these for now!

Reply:
They might be at some point. No harm in leaving it there.